docs/source/usage/deployment.rst

```diff
@@ -135,7 +160,40 @@ This is where we take everything we've done up to this point and start the deplo
+     18. Once the API Gateway has been created, copy the URL into your webhook application's configuration as the endpoint to post events to begin sending events to the AWS infra that was deployed using this framework.
+
+.. toggle-header::
+    :header: **2. Using a Single Source Repo (GitHub)**
+   
+     1. Pull the empty Github repository created earlier in section `Getting Started` to a local directory, then add/move all the cloned and updated framework code to this repository. Add and commit all changes, then push.
+     2. Go to “Services -> CloudFormation” on the AWS Console.
+     3. On top right, click “Create stack (with new resources)”
+     4. Select “template is ready” on the first option. In “specify template” menu, select “upload a template file”
+
+       .. toggle-header::
+           :header: **2. Create stack from Template:**
+       
+             Example screenshot for creating a stack from the template
+       
+             .. image:: ../images/create_stack.png
+                :width: 400
+                :alt: AWS Cloudformation Console Create Stack
+
+     5. Browse the filesystem, and select the CloudFormation template you wish to use under the “templates” directory of the cloned and updated framework code. Assuming no syntax errors, click next.
+     6. For the stack name, enter something descriptive, like: <project>-<service> (e.g., mozdef-ef-zoom, see the example image below for steps 8 through 14)
+     7. For stack parameters, enter the values decided in "Getting Started" Step 2.
+     8. For service, enter your <service> name that you determined in the "Getting Started" section Step 2.
+     9. For environment, choose “dev”, "staging", or "prod" according to the environment you are working out of.
+     10. In the TOKEN_ARN field, you'll need to enter your token arn to correctly map the iam permissions for this resource.
+     11. An S3 utility bucket will be created for AWS CodePipeline to store artifacts. The bucket name will match the parameters you created for your stack name in step 8 and the environment in step 11 (e.g., <stackname>-<environment>-utility)
+     12. For source configuration, enter the name of the Github repo housing the code, in the following format: `owner/repository/branch`.
+     13. Under stack options, add a tag with key: Project and value: <project>-<environment>. Click Next
+     14. On the review step, check the box under “Capabilities” saying “I acknowledge that AWS CloudFormation might create IAM resources with custom names.”.
+     15. Click Create Stack. On the Cloudformation page, check the stack creation status. It should deploy the pipeline stack successfully.
+     16. Once the API Gateway has been created, copy the URL into your webhook application's configuration as the endpoint to post events to begin sending events to the AWS infra that was deployed using this framework.
+
+.. toggle-header::
+    :header: **3. Using Multiple Source Repos (CodeCommit + Github)**
```

Comment:
I stopped documenting this before we decide whether or not we want to create the CodeCommit repo manually beforehand or let the CF template create it for us (as is). Once we decide that I am happy to provide steps here.

Reply:
I believe this was resolved by creating the repo first, instead of the CF doing it.